sections/futures/MarketDetails/utils.ts

```diff
@@ -1,11 +1,33 @@
 import { Synths } from 'constants/currency';
 
-const markets = [Synths.sETH, Synths.sBTC, Synths.sLINK] as const;
+const markets = [
+	Synths.sETH,
+	Synths.sBTC,
+	Synths.sLINK,
+	Synths.sSOL,
+	Synths.sAVAX,
+	Synths.sMATIC,
+	Synths.sAAVE,
+	Synths.sUNI,
+	Synths.sEUR,
+	'sXAU',
+	'sXAG',
+	'sWTI',
```

Comment:
hardcoding these as Synths has no reference to them for now. this prevents price feeds from defaulting to sETH and displaying the eth price -- instead it will show as `-` as listed in the requirements